rqt_gui/package.xml

```diff
@@ -16,18 +16,14 @@
 
   <build_depend version_gte="0.3.0">qt_gui</build_depend>
 
-  <exec_depend>ament_cmake_python</exec_depend>
+  <exec_depend>ament_index_python</exec_depend>
   <exec_depend>python_qt_binding</exec_depend>
   <exec_depend>python3-catkin-pkg-modules</exec_depend>
   <exec_depend>qt_gui</exec_depend>
   <exec_depend>rclpy</exec_depend>
-  <exec_depend version_gte="0.3.0">qt_gui</exec_depend>
```

Comment:
The version dependency should be kept on the remaining exec dependency.

Reply:
Fixed